src/app/page.js

```diff
@@ -1,9 +1,135 @@
-import Image from 'next/image'
-
+"use client"
+import Image from 'next/image';
+import { useState } from "react";
+import { useRouter } from 'next/navigation';
 export default function Home() {
+  const [searchTerm, setSearchTerm] = useState('');
+  const router = useRouter();
```

Comment:
am yet to see **routers** in **next js**, as they have introduced a very nice way to handle **routes**.